### ISSUE TITLE
Bug: Adding a highlight from profile page doesn't immediately add the highlight

### ISSUE THREAD

**Issue by adiati98** [open-sauced/app#3128]
### Describe the bug

When I added a highlight from the "Highlights" tab at my profile page, it didn't add the highlight immediately to the list. But it did show when clicking the "Highlights" tab on the navbar. However, it finally showed on my profile a half hour later.

### Steps to reproduce

1. Go to profile page
2. Create a highlight at the "Highlights" tab.
3. You won't see the newly created highlight on the list. But there is a notification that highlight is successfully added.
4. Click the "Highlights" tab on the navbar.
5. You will see the newly created highlight here.
6. Back to your profile and click "Highlights" tab.
7. You still won't see the highlight.
8. Get back to it around half hour later.
9. You will now see the highlight.

### Screen Recording

![adding highlight from profile](https://github.com/open-sauced/app/assets/45172775/3d210cc4-4640-49de-9b90-e3ad990cf04f)

### Screenshots

![Screenshot 2024-04-05 051323](https://github.com/open-sauced/app/assets/45172775/85ca35a3-751f-4ff0-9384-44a7f2570bfb)

### Browsers

Chrome

### Additional context (Is this in dev or production?)

production

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

**Comment by shubhamchasing** [comment]
Yeah this bug exists, I would like to try to solve this, please tag me when `needs triage` label will be removed.

.take

**Comment by nickytonline** [comment]
All yours @shubhamchasing!

**Comment by shubhamchasing** [comment]
The issue is from the backend. 

The get API of user highlights  https://beta.api.opensauced.pizza/v2/users/${username}/highlights?page=1 does not have the recently added highlight from the user profile section for at least 10 to 15 minutes.

and there is no issue with the get API of all  https://beta.api.opensauced.pizza/v2/highlights/list?page=1&limit=10

Currently, I do not have much exposure to the backend
